src/TLuaInterpreter.cpp

```diff
@@ -14387,13 +14385,21 @@ int TLuaInterpreter::getServerEncodingsList(lua_State* L)
 {
     Host& host = getHostFromLua(L);
 
+    // don't leak if we're using a Mudlet or a Qt-supplied codec to Lua
+    auto sanitizeEncoding = [] (auto encodingName) {
```

Comment:
If we are doing this, do we not need the *same* sanitization for the results from the `getServerEncodingsList()` function? :confused: